langlib/lang.decimal/src/main/ballerina/decimal.bal

```diff
@@ -18,6 +18,16 @@ import ballerina/jballerina.java;
 
 # Returns the sum of zero or more decimal values.
 #
+# ```
+# decimal:sum(1.2, 2.3, 3.4) ⇒ 6.9
+#
+# decimal[] scores = [11.1, 22.2, 33.3];
+# decimal:sum(...scores) ⇒ 66.6
+#
+# decimal d = 21.2;
+# d.sum(10.5, 21, 32.4) ⇒ 85.1
```

Comment:
```suggestion
# 21.2.sum(10.5, 21, 32.4) ⇒ 85.1
```

Reply:
Please refer to this comment. https://github.com/ballerina-platform/ballerina-lang/pull/38905#discussion_r1048012779